cms/release_calendar/migrations/0002_create_releasecalendarindex.py

```diff
@@ -0,0 +1,47 @@
+from django.db import migrations
+from django.utils import timezone
+
+
+def create_index_page(apps, schema_editor):
+    # Get models
+    ContentType = apps.get_model("contenttypes.ContentType")
+    IndexPage = apps.get_model("release_calendar", "ReleaseCalendarIndex")
+    Page = apps.get_model("wagtailcore.Page")
+    Locale = apps.get_model("wagtailcore.Locale")
+
+    # Create content type for the model
+    index_content_type, created = ContentType.objects.get_or_create(
+        model="releasecalendarindex", app_label="release_calendar"
+    )
+
+    home_page = Page.objects.get(url_path="/home/")
+    now = timezone.now()
+
+    # Create a new page
+    IndexPage.objects.create(
+        title="Release calendar",
+        draft_title="Release calendar",
+        live=True,
+        first_published_at=now,
+        last_published_at=now,
+        slug="releases",
+        path="%s00%02d" % (home_page.path, home_page.numchild + 1),
+        content_type=index_content_type,
+        depth=home_page.depth + 1,
+        url_path=f"{home_page.url_path}releases/",
+        locale=Locale.objects.first(),  # cannot use Locale.get_default(), but since we only have one, this works
```

Comment:
Nit: Worth using `.get` to enforce there's only one?
```suggestion
        locale=Locale.objects.get(),  # cannot use Locale.get_default(), but since we only have one, this works
```

Reply:
leaving this so things don't break when add i18n